test/models/scheduled-notification.model.test.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../support/database.js')
+const EventHelper = require('../support/helpers/event.helper.js')
+const ScheduledNotificationHelper = require('../support/helpers/scheduled-notification.helper.js')
+const EventModel = require('../../app/models/event.model.js')
```

Comment:
I'll get the ESLint plugin added soon I promise!

```suggestion
const EventHelper = require('../support/helpers/event.helper.js')
const EventModel = require('../../app/models/event.model.js')
const ScheduledNotificationHelper = require('../support/helpers/scheduled-notification.helper.js')
```